### DISCUSSION TITLE
Roam Research Obsidian Notes- Graph Visualization Spider Web Idea

### DISCUSSION THREAD

**Discussion by morningstar41131411811717116112213** [Antonoko/Windrecorder#168] [Ideas 我有个想法]
I love the lightbox and word cloud idea of memories and recordings. 

Thought a RoamResearch Graph visualization inspiration (Obsidian Second Brain too) would be a very awesome idea to look into. 

Let me know what you guys think. 

https://github.com/org-roam/org-roam-ui
https://roamresearch.com/#/app/help/graph

🕸️

Blessings : )

**Comment by morningstar41131411811717116112213** [comment]
Also maybe an idea to add custom theme background

Uploading an image or inserting a Url for example

I would like to use this as my background be it transparent or blurred out example:

![QmVJacXe59nqRB9JkJbNwWzcNjxTaHbzzqS8ZS4cXYTK19-min](https://github.com/yuka-friends/Windrecorder/assets/71581407/3ee032d4-2487-482f-b3f9-d7058479dd96)

**Comment by ASC8384** [comment]
The visualization concepts for both Obsidian and RoamResearch are built around nodes and the connections between them. Could you give some examples of what the nodes might look like for Windrecorder?

**Reply by morningstar41131411811717116112213** [reply]
For Windrecorder, visualizing nodes and their connections can help organize and navigate recorded data efficiently. Each recording session can be represented as a node, including details like date, time, duration, and tags. These nodes could be linked to user nodes representing who created or accessed the recording, and context nodes capturing metadata like applications or websites active during the recording.

Other nodes could represent specific events or highlights within recordings, such as key moments or bookmarks. This approach allows for a visually organized system, making it easier to manage, search, and analyze recordings, similar to the node-based visualization in tools like Obsidian and RoamResearch.

**Comment by Antonoko** [comment]
Thanks for your feedback! The knowledge map is a very interesting idea. I feel like we can think about how to build a connected overview map around the reading activities we are exposed to and the information we collect. More research may be needed. Like, is it clustered and organized based on similar content, or organized according to different application activities? See how to find interesting connections in information.

The custom theme background should be possible to implement it through simple CSS modifications. Since there are relevant requirements, it may be added as a hidden small function in the future.

**Reply by morningstar41131411811717116112213** [reply]
Thank you guys!

Yeah this is just off the top of my head. 
Hopefully I can get more time, we could possibly brainstorm a concreate way to approach it. 

Speaking of nodes, are you familiar with Infranodus?
That may be a great inspiration to approach, maybe nodes to be used would be dates, times, timestamp notes, and lightbox summaries?
List goes on hehe. 
Take care and here's to an infinite future!

**Comment by morningstar41131411811717116112213** [comment]
For Windrecorder, visualizing nodes can help organize and navigate recorded sessions efficiently. Each recording session could be a node with details like date, time, and tags, linked to user nodes representing who created or accessed the recording. Context nodes might capture metadata like active applications or websites during the recording, and event nodes could mark specific highlights or key moments within recordings. This can be visualized as a graph similar to RoamResearch, where nodes are interconnected, creating an intuitive, organized system that makes it easier to manage, search, and analyze recordings.

**Reply by morningstar41131411811717116112213** [reply]
Also look into multilingual OCR

See the example for other languages:

https://sourceforge.net/projects/capture2text/files/Dictionaries/
https://capture2text.sourceforge.net/

**Reply by morningstar41131411811717116112213** [reply]
sup my guys!

were you able to look into the OCR packages above right now? ^


Would love for an italian or portuguese one firstly helps with the left tor right languages i am using with the Turkish OCR etc etc. 

But any will do. 

Thank you so much again!

**Reply by Antonoko** [reply]
roger that, we can see if it is convenient to embed as extension under this PR in the future (https://github.com/yuka-friends/Windrecorder/pull/146) (still work in process to abstract to a more common interface). 

Regarding the existing [Windows built-in language pack](https://support.microsoft.com/en-us/windows/language-packs-for-windows-a5094319-a92d-18de-5b53-1cfc697cfca8) which conclude Turkish, I would like to know if it is usable? (after installation, it should be available as a choice of OCR language) (is it because of insufficient recognition accuracy or the need to recognize multiple languages ​​at the same time that need to take advantage of external libraries?

**Reply by morningstar41131411811717116112213** [reply]
The latter my friend. 
In a way I know Rewind allows toggling between multilingual support atleast recognizes in real time both, even one over the other. 
It's just I'm doing so many things at once, and specifically for language learning this would be so fruitful. 

Thanks my man.

**Reply by morningstar41131411811717116112213** [reply]
Also just another suggestion. 
I love finding the keywords and search querys on my own lol it's like playing Waldo. 

But just as a future option, having the choice to decide between toggling the search highlighted or not. 

Similar to Rewind.Ai, the OCR or search engine is able to highlight your request be it text as well as in the video or frame chosen. 

That and the ability to visit the url or link (see Infr or Unlost) when clicking on it, to restore browsing mark/left off at. 

But yeah I love it so much. Keep it up fam.

**Comment by Antonoko** [comment]
just found an application that can summarize browsing history into Obsidian-like Notes-Graph:
https://www.producthunt.com/posts/active-recall

**Reply by morningstar41131411811717116112213** [reply]
I've tried this before bro. it's okay lol. 

I suggest you look into Infranodus. It's higher end, but it's worth it of course.

**Comment by morningstar41131411811717116112213** [comment]
Yo bros look into this Remind repository.  https://github.com/DonTizi/ReMind


A summarization and Jarvis like feature of the recalling process. 
More cherries on top.
